pkg/controller/controller_instance_test.go

```diff
@@ -186,18 +179,23 @@ func TestReconcileInstanceNonExistentServicePlan(t *testing.T) {
 	assertNumberOfBrokerActions(t, brokerActions, 0)
 
 	actions := fakeCatalogClient.Actions()
-	assertNumberOfActions(t, actions, 1)
-
-	// There should only be one action that says it failed because no such class exists.
```

Comment:
@MHBauer I'll move the comment down, but I'll need to change it because we're checking to make sure the error represents the fact that the service plan doesn't exist